src/cursor.jl

```diff
@@ -369,38 +335,30 @@ isvirtual(c::CLCursor) = isvirtual(c.cursor)
 #
 
 """
-    spelling(c::CXCursor) -> String
-    spelling(c::CLCursor) -> String
+    spelling(c::Union{CXCursor,CLCursor}) -> String
 Return a name for the entity referenced by this cursor.
 """
-function spelling(c::CXCursor)
-    GC.@preserve c begin
```

Comment:
This preserve seems necessary if either the two input types use finalizer to manage the memory related to any of the pointers

It would not be necessary if `cxstr` memory is not bound to `c`

Reply:
And same with most of the ones that preserve c below

Reply:
According to https://clang.llvm.org/doxygen/CXString_8h_source.html：
> when the ownership of that string might differ from one call to the next

I guess this implies `cxstr` memory is not bound to `c`.